modules/betsy/betsy_common_inc.glsl

```diff
@@ -0,0 +1,2 @@
+#define min3(a, b, c) min(a, min(b, c))
+#define max3(a, b, c) max(a, max(b, c))
```

Comment:
Given how little those are used, I'd argue for dropping this file and just using `min(a, min(b, c))` etc. in the 3 locations they're needed:
```
~/.../modules/betsy (setup_debug_symbols)$ rg min3
CrossPlatformSettings_piece_all.glsl
1:#define min3(a, b, c) min(a, min(b, c))

bc4.glsl
57:     minVal = min3(srcPixel.x, srcPixel.y, srcPixel.z);
~/.../modules/betsy (setup_debug_symbols)$ rg max3
CrossPlatformSettings_piece_all.glsl
2:#define max3(a, b, c) max(a, max(b, c))

bc4.glsl
58:     maxVal = max3(srcPixel.x, srcPixel.y, srcPixel.z);

bc1.glsl
144:    float magn = max3(abs(vF.r), abs(vF.g), abs(vF.b));
```